frontend/src/components/SearchForm.js

```diff
@@ -3,7 +3,7 @@ import '../sass/components/searchform.scss';
 import '../sass/base/typography.scss';
 import icon from '../assets/search-icon.png';
 
-const SearchForm = () => (
+const SearchForm = ({radioDifficulty, setRadioDifficulty}) => (
```

Comment:
I think this component should control its own state rather than drilling it down from the parent. This is because the parent `Main` is not really using the state for anything. So, please make this component stateful and not the `Main` component.

Reply:
You are right, fixed.